data/src/main/java/org/oppia/android/data/backends/gae/NetworkModule.kt

```diff
@@ -17,7 +17,8 @@ import javax.inject.Singleton
  * Sample resource: https://github.com/gahfy/Feed-Me/tree/unitTests
  */
 @Module
-class NetworkModule {
+class
+NetworkModule {
```

Comment:
Should be on the same line.

Reply:
Ah, I saw this as well but it didn't show up on my AS until now. Odd